### ISSUE TITLE
[TW-388] please make TASK_RCDIR customizable

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#423]
_Jakub Wilk on 2014-01-04T17:14:54Z says:_ 

 The attached patch adds TASK_RCDIR as cmake configuration variable.

Background:

Debian has a policy that packages must not require the existence of any files in /usr/share/doc/ in order to function, so we change TASK_RCDIR to /usr/share/task/.

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2014-01-04T17:14:54Z
Modified: 2014-02-09T01:36:04Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2014-01-04T17:38:01Z says:_ 

 Thanks.

**Comment by taskwarrior** [comment]
_Elias Probst on 2014-01-22T23:14:29Z says:_ 

 This patch was missing something: _doc/CMakeLists.txt_ still used _TASK_DOCDIR_ for the directory _doc/rc_, so the content of this directory still went to the wrong target path.

The attached patch fixes this issue.

Could someone with the required permissions either re-open this issue or directly apply (after reviewing it) this patch to master + 2.4.0?

**Comment by taskwarrior** [comment]
_Elias Probst on 2014-01-22T23:14:29Z says:_ 

 Attachment issue-1473-rcdir-fix.patch has been added with description: Fix usage of TASK_DOCDIR → TASK_RCDIR in doc/CMakeLists.txt

**Comment by taskwarrior** [comment]
_Elias Probst on 2014-01-22T23:32:08Z says:_ 

 Missed this in my previous comment: should also be applied to the 2.3.0 branch.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2014-01-22T23:34:08Z says:_ 

 > Missed this in my previous comment: should also be applied to the 2.3.0 branch.

2.3.0 is released and merged to master, so we don't touch it any more.

We are now working in the unstable 2.4.0 branch, and in the event of an emergency, we will cherry-pick fixes into a 2.3.1 branch and release that.

Meanwhile thank you for the patch - it is applied to 2.4.0 and will be pushed shortly.

**Comment by pbeckingham** [comment]
[TW-388_add-TASK_RCDIR-configuration-variable.patch.txt](https://github.com/GothenburgBitFactory/taskwarrior/files/1755599/TW-388_add-TASK_RCDIR-configuration-variable.patch.txt)
[TW-388_issue-1473-rcdir-fix.patch.txt](https://github.com/GothenburgBitFactory/taskwarrior/files/1755600/TW-388_issue-1473-rcdir-fix.patch.txt)
